src/components/MessagesList/MessagesList.vue

```diff
@@ -805,16 +805,17 @@ export default {
 					// already loading, don't do it twice
 					return
 				}
-				if (scrollTop === 0) {
-					this.displayMessagesLoader = true
-				}
+				this.displayMessagesLoader = true
 				await this.getOldMessages(false)
 				this.displayMessagesLoader = false
 
 				if (this.$refs.scroller.scrollHeight !== scrollHeight) {
-					this.$refs.scroller.scrollTo({
-						top: this.$refs.scroller.scrollTop + this.$refs.scroller.scrollHeight - scrollHeight,
-					})
+					// If was scrolled to the top, scroll for added height, otherwise browser handles it natively
+					if (scrollTop === 0) {
```

Comment:
I think you meant this.$refs.scroller.scrollTop here ? 
Because if `scrollTop  === 0` , scrollHeight won't change as we reached the very top.

Reply:
Both should be equal 0, but maybe, the actual value would be correct to use